openrarity/metrics/ic.py

```diff
@@ -40,6 +39,19 @@ def information_content(
 
 
 def calculate_entropy(attr_stats: list[AttributeStatistic]) -> float:
+    """
+    Entrophy is calculated using cumulative sum of the products of `metric.probability` and `metric.information`.
```

Comment:
Spelling. And again layman terms. What _IS_ entropy? 
> Information Entropy is the sum of the product of the probability and information content for each attribute. 
Then describe what it is as a concept as well. You will need to research this.